core/server/utils/cached-image-size-from-url.js

```diff
@@ -19,7 +19,7 @@ function getCachedImageSizeFromUrl(url) {
 
     // image size is not in cache
     if (!imageSizeCache[url]) {
-        return getImageSizeFromUrl(url, 6000).then(function (res) {
+        return getImageSizeFromUrl(url, 5000).then(function (res) {
```

Comment:
Rather than passing the timeout here, and also having a default of 10000 inside of image-size-from-url.js, I would prefer to have it defined in only one place (image-size-from-url.js) and also be pulled from config so that we can control this without releasing if needed.